packages/muelu/research/regionMG/examples/structured/Driver_Structured_Regions.cpp

```diff
@@ -839,7 +838,7 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
         /////////////////////////////////////////////////////////////////////////
 
         //      printRegionalObject<Vector>("regB 2", regB, myRank, *fos);
-        vCycle(0, numLevels, maxCoarseIter, maxRegPerProc,
```

Comment:
Same as previous comment, after this change `const int maxCoarseIter = 100;` on line 775 becomes unused.